docs/markdown/android/overlaypanel.md

```diff
@@ -0,0 +1,115 @@
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/41/a6/36/41a6369d262d24bf04d08ee32beef508.png" alt="examples of three types of overlay panels: a Full overlay panel, a Partial resizable overlay panel and an Action overlay panel"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
+      - Performing an optional sub-task within a larger task
+      - Presenting help info while maintaining the current page and its context
+      - Requesting minimal amounts of information froma user (1-2 fields only)
+      - Capturing user’s full attention for something important
+  </Group>
+  <Group>
+    <Dont title="When not to use" />
+      - Displaying system errors or notices. Consider a [Callout](callout) instead
+      - Any time a separate, designated URL is desired
+      - Any action that should not interrupt users from their current work stream
+      - On top of another overlay panel, as this can create usability issues and confusion
+  </Group>
+</TwoCol>
+
+## Mobile best practices
+
+- OverlayPanels should be triggered by the users action, like tapping on a button.
+- Use an overlay panel for non-imersive content and simple tasks.
+- OverlayPanels always sit in front of other UI elements.
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/84/d4/0d/84d40d290128608db361a276a586da87.png" alt="a partial overlay panel with a close button and a grabber to allow for multiple ways to close and collapse"/>
+    <Do title="Do" />
+    Provide a collapse or a close affordance&mdash;multiple if possible. This could include an X, back arrow, grabber or close button.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/5d/3a/20/5d3a20657045824cb4b7d89263b6ec30.png" alt="a partial overlay panel with a button in the top right corner of the header"/>
+    <Dont title="Don't" />
+    Include Buttons in the header of the OverlayPanel as this can lead to sizing and internationalization problems.
+  </Group>
+</TwoCol>
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/06/e9/6f/06e96f0193da3a32a25d7f1174d5dfa1.png" alt="a partial overlay panel with a grabber to help resize"/>
+    <Do title="Do" />Utilize a grabber for Partial overlay panels that are able to be resized.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/62/a4/fa/62a4fa0befb32c836519b6d9f1af5f28.png" alt="a full overlay panel with rounded corners"/>
+    <Dont title="Don't" />
+    Round the corners of full overlay panels. Only Partial and Action overlay panels should have rounded corners.
+  </Group>
+</TwoCol>
+
+## Variants
+
+### Size
+
+OverlayPanels are available in three sizes.
+
+1. **Full overlay panel**
```

Comment:
typo: "OverlayPanel" (here and throughout this section)

Reply:
UNDOING